lib/csv.rb

```diff
@@ -928,7 +928,8 @@ def initialize(message, line_number)
     symbol:   lambda { |h|
       h.encode(ConverterEncoding).downcase.gsub(/[^\s\w]+/, "").strip.
                                            gsub(/\s+/, "_").to_sym
-    }
+    },
+    symbol_raw: lambda { |h| h.to_sym }
```

Comment:
Could you convert it to `ConverterEncoding` like others?

```suggestion
    symbol_raw: lambda { |h| h.encode(ConverterEncoding).to_sym }
```

Reply:
I understood it. Thanks.